guides/source/active_job_basics.md

```diff
@@ -389,6 +389,25 @@ class GuestsCleanupJob < ApplicationJob
 end
 ```
 
+### Retrying or Discarding failed jobs
+
+It's also possible to retry or discard a job if an exception is raised during execution.
+For example:
+
+```ruby
+class RemoteServiceJob < ActiveJob::Base
+  retry_on CustomAppException # defaults to 3s wait, 5 attempts
+
+  discard_on Net::OpenTimeout
```

Comment:
I think that discarding `Net::OpenTimeout` is a bad example. Generally `Net::OpenTimeout` might be raised when the remote service is down, and it will be recovered by retrying.
Can you use `ActiveJob::DeserializationError` like the original doc?

Reply:
done, ready for review thx